saliency_metrics/metrics/insertion_deletion/insertion_deletion_metric.py

```diff
@@ -0,0 +1,89 @@
+from typing import Any, Dict, List
+
+import numpy as np
+import torch
+from scipy.integrate import trapezoid
+from torchvision.transforms import GaussianBlur
+
+from saliency_metrics.metrics.build_metric import ReInferenceMetric
+from saliency_metrics.models.build_classifier import build_classifier
+from .insertion_deletion_perturbation import ProgressivePerturbation
+from .insertion_deletion_result import InsertionDeletionResult
+
+
+class InsertionDeletion(ReInferenceMetric):
+    def __init__(
+        self,
+        classifier_cfg: Dict,
+        forward_batch_size: int = 128,
+        perturb_step_size: int = 10,
+        sigma: float = 5.0,
+        summarized: bool = False,
+    ) -> None:
+
+        self._result = InsertionDeletionResult(summarized)
+
+        self.classifier = build_classifier(classifier_cfg)
+        # TODO - check
+        # self.classifier.eval()
```

Comment:
Please use the utility functions in `saliency-metrics` to freeze the parameters and turn on the `eval` mode.